### PR TITLE
Added about page for project_MDMA

### PR DESCRIPTION
**Pull request by Anjali0407-git**

### CODE REVIEW

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
This is all nearly perfect. Two minor things:
1. Let's get all the contributors to the original project listed. We can use the GH commit and PR history to get all of them, and we be able to get their GH profile links from there, too.
2. Please add this to the main portfolio page under the External Projects section.

**Comment by Anjali0407-git** [comment]
Sure Daniel, I would add all contributors along with their GH profiles.
Also I would add the project to Portfolio page.

File: docs/portfolio.md [hunk 1]
```diff
@@ -15,47 +15,48 @@ We are especially interested in projects supporting researchers, focused on rese
...
-**[Where's Religion Desktop](project_wheres_religion_desktop/about.md)** - Where's Religion Desktop is an enhancement on the mobile version which will have extended features.  
-  
-**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
+**[Shelter Volunteers](project_shelter_volunteers/about.md)** - Simplify the process of scheduling work shifts for volunteers, and to give homeless shelters visibility into their upcoming staffing.
+
+**[Speech Tagging](project_saltify/about)** - This software transcribes an audio sample into a written format that is accepted by SALT software.
+
+**[Where's Religion Desktop](project_wheres_religion_desktop/about.md)** - Where's Religion Desktop is an enhancement on the mobile version which will have extended features.
+
+**[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.
```
Comment (Anjali0407-git) [inline docs/portfolio.md:47]: @kungfuchicken the above changes are just trivial and are added by my Prettify extension.

File: docs/project_MDMA/about.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  - Siri Chandana Garimella [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/SiriChandanaGarimella)
+  - Manohar Reddy YRL [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yrlmanoharreddy)
+  - Chirag Gupta [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Chirag2x)
+  - Abhi Stephen Rokkam [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Abhi-Stephen)
+  - Krishnakanth Burugu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/krishnakanth-slu)
+  - Brijitha Tialu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Brijitha1609)
+  - Deepak PVSN [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/deepak-pvsn)
+  - Maryam Moshrefizadeh [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Moshrefi)
+  - Lalith Adithya Reddy Avuthu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/alar12)
```
Comment (Anjali0407-git) [inline docs/project_MDMA/about.md:26]: @kungfuchicken added the details of all contributors to the main branch

File: docs/project_MDMA/about.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+custom_edit_url: null
+---
+
+## Overview
+
+The Material Donor Mutual Assist project by BWorks is designed to streamline the process of managing donations, providing a transparent view into the journey of each donated item. This initiative allows donors to track their contributions from the moment they are donated to their ultimate use, offering them a detailed view of how their items are making a difference. By keeping donors informed about the status and use of their donations, the project fosters a sense of trust and community, encouraging continued support and involvement. The system simplifies the administrative tasks involved in updating and managing donations, making it easier to attract and retain donors who are motivated by seeing the tangible impact of their contributions.
+
+### Information
+
+- **Source Code:** [https://github.com/slu-csci-5030/material-donor-mutual-assist/](https://github.com/slu-csci-5030/material-donor-mutual-assist/) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/slu-csci-5030/material-donor-mutual-assist/)
```
Comment (kungfuchicken) [inline docs/project_MDMA/about.md:13]: This repo will not be available to outsiders. We should clone this into the OSS organization. I'll merge it for now, but let's follow up with an edit once that's done.

**Review by kungfuchicken — APPROVED** [review]
follow up todo: clone the old repo into the OSS org, and change the `**Source Code:**` link.

**Pull request merged**